CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 21.55.0
```

Comment:
What do you think about adding [`govuk-frontend` bumps](https://github.com/alphagov/govuk_publishing_components/pull/1549) to the changelog? We haven't done this previously, but we often ask ourselves with which version was the update rolled out, and it often affects how components look like or behave.

Reply:
I think that's a good idea. I'll add it.

Reply:
Added.